src/java.base/share/classes/javax/crypto/EncryptedPrivateKeyInfo.java

```diff
@@ -57,17 +57,18 @@
 
 public class EncryptedPrivateKeyInfo {
 
-    // the "encryptionAlgorithm" field
+    // The "encryptionAlgorithm" field. If this object is created by
+    // {@link #EncryptedPrivateKeyInfo(AlgorithmParameters, byte[])}
+    // with an uninitialized params, it's stored here and algid is null.
```

Comment:
A bit confused by the words "it's stored here" if it is always null for the case where the params are uninitialized. How about: "... with an uninitialized params, algid is null."

Reply:
ok but change "store" to "stored" and "algid" to "algid is set to null".